service/storage/operations_internal_test.go

```diff
@@ -984,7 +989,7 @@ func TestSaveAndRetrieve_Guarantee(t *testing.T) {
 			codec: codec,
 		}
 
-		err := db.Update(l.SaveGuarantee(mocks.GenericGuarantee(0)))
+		err := db.Update(l.SaveGuarantee(guar))
```

Comment:
In this case, I would definitely write it out as `guarantee`. For consistency's sake, we should use `collID`, even when it's about guarantees, because collection guarantees don't have an ID of their own, they always refer to a collection. I know this is not consistent in some places of the Flow Go code.